src/dredd-transactions.coffee

```diff
@@ -1,36 +1,22 @@
 
 parse = require('./parse')
-compileFromApiBlueprintAst = require('./from-api-blueprint-ast/compile')
-compileFromApiElements = require('./from-api-elements/compile')
+compileFromApiElements = require('./compile')
```

Comment:
Filed https://github.com/apiaryio/dredd-transactions/issues/27 as a tombstone for this piece of tech debt.